frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentHeader/FeatureOverviewEnvironmentMetrics/FeatureOverviewEnvironmentMetrics.tsx

```diff
@@ -0,0 +1,90 @@
+import type { IFeatureEnvironmentMetrics } from 'interfaces/featureToggle';
+import { calculatePercentage } from 'utils/calculatePercentage';
+import PercentageCircle from 'component/common/PercentageCircle/PercentageCircle';
+import { PrettifyLargeNumber } from 'component/common/PrettifyLargeNumber/PrettifyLargeNumber';
+import { styled } from '@mui/material';
+
+interface IFeatureOverviewEnvironmentMetrics {
+    environmentMetric?: IFeatureEnvironmentMetrics;
+    disabled?: boolean;
+}
+
+const StyledContainer = styled('div')({
+    marginLeft: 'auto',
+    display: 'flex',
+    alignItems: 'center',
+});
+
+const StyledInfo = styled('div')(({ theme }) => ({
+    marginRight: theme.spacing(1.5),
+    display: 'flex',
+    flexDirection: 'column',
+}));
+
+const StyledPercentage = styled('p')(({ theme }) => ({
+    color: theme.palette.primary.main,
+    textAlign: 'right',
+    fontSize: theme.fontSizes.bodySize,
+}));
+
+const StyledInfoParagraph = styled('p')(({ theme }) => ({
+    fontSize: theme.typography.body2.fontSize,
+    textAlign: 'right',
+    [theme.breakpoints.down('xl')]: {
+        display: 'none',
+    },
+}));
+
+const StyledPercentageCircle = styled('div')(({ theme }) => ({
+    marginRight: theme.spacing(1),
+    [theme.breakpoints.down(500)]: {
+        display: 'none',
+    },
+}));
+
+const FeatureOverviewEnvironmentMetrics = ({
+    environmentMetric,
+    disabled = false,
+}: IFeatureOverviewEnvironmentMetrics) => {
+    if (!environmentMetric) return null;
+
+    const total = environmentMetric.yes + environmentMetric.no;
+    const percentage = calculatePercentage(total, environmentMetric?.yes);
+
+    if (
+        !environmentMetric ||
+        (environmentMetric.yes === 0 && environmentMetric.no === 0)
+    ) {
+        return null;
+    }
+
+    return (
+        <StyledContainer>
+            <StyledInfo>
+                <StyledPercentage>{percentage}%</StyledPercentage>
+                <StyledInfoParagraph>
+                    The flag has been requested{' '}
+                    <b>
+                        <PrettifyLargeNumber value={total} /> times
+                    </b>
+                    <br />
```

Comment:
This is a personal thing, but I don't `br` elements. I know we use them lots of places, but I'd rather wrap it in a span and set display to block or something. But this isn't important and you're free to disagree.

Reply:
Replaced with by doing it with CSS